.env.example

```diff
@@ -0,0 +1,55 @@
+#=======================
```

Comment:
The number of required env variables has been reduced dramatically (https://github.com/paritytech/parity-processbot/pull/329). I think you won't have to include this file anymore.

Reply:
Yeah, mentioned this above https://github.com/paritytech/parity-processbot/pull/320#discussion_r728555608.

Looks like you've merged in this file